tests/test_fair_utils.py

```diff
@@ -57,3 +59,143 @@ def test_get_weights():
     )
     np.testing.assert_allclose(get_weights([1]), np.array([1]))
     np.testing.assert_allclose(get_weights([0]), np.array([1]))
+
+
+def test_quantile_weight_quantilereg_dim():
```

Comment:
@JMGilbert thanks for adding these tests! I think we also need to test real values (not just 1's). And we should talk about how to test rff-sp based data.

Reply:
Consider this for a future PR